pravega-cli/src/test/java/io/pravega/tools/pravegacli/ControllerCommandsTest.java

```diff
@@ -37,9 +37,11 @@ public static void setup() throws Exception {
         SETUP_UTILS.startAllServices();
         STATE.set(new AdminCommandState());
         Properties pravegaProperties = new Properties();
-        pravegaProperties.setProperty("cli.controllerRestUri", SETUP_UTILS.getControllerRestUri().toString());
+        pravegaProperties.setProperty("cli.controllerRestUri", SETUP_UTILS.getControllerRestUri().toString().substring(7));
```

Comment:
It's not clear looking at this code alone why the substring is needed. Please add an explaining comment.

Reply:
Hmm... then you could simply replace `http://` with an empty string and the code itself shall make the intent explicit.

Reply:
I've added a comment explaining the same.